build/helper/metadata_add_all.py

```diff
@@ -691,6 +699,7 @@ def _compare_dicts(actual, expected):
                 'direction': 'in',
                 'enum': None,
                 'name': 'channelName',
+                'python_name': 'name',
```

Comment:
We need a couple more unit tests for your changes:
1. `name = channel_name` and `python_name` is not set
1. `name = channel_name` and `python_name = name`
1. `name = name` and `python_name = channel_name`

Reply:
I'm a bit confused. All the other unit tests in this class have `python_name` == `name` (with the caveat that one is in camel case and the other in snake case). I added this one so that we'd also have a `python_name` != `name`. So I don't really understand what the other combinations in the examples you listed are, since I'm not sure what `channel_name` represents in your examples.

Also, I don't think it's valid to have a unit test in which `python_name` isn't set -- based on what you had said a couple of days ago ("By design the expanded metadata should contain all the keys in all the instances but `python_name_override` is only added to some").

Reply:
I've reverted the changes I had made to this unit test, since I'm no longer using `python_name_override` in `_add_is_repeated_capability()`. When I create a separate PR with the `is_repeated_capability` I'll update the test accordingly.